src/main/webapp/app/app.module.ts

```diff
@@ -15,6 +15,7 @@ import { PageRibbonComponent } from './layouts/profiles/page-ribbon.component';
 import { ActiveMenuDirective } from './layouts/navbar/active-menu.directive';
 import { ErrorComponent } from './layouts/error/error.component';
 import { LoginModule } from './login/login.module';
+//import { HomeModule } from './home/home.module';
```

Comment:
If this module is not needed delete this line (and line 31)

Reply:
you're right . i will remove it